src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,118 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.ErrorCode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatusFrame;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.config.Config;
+
+public class Arm extends SubsystemBase {
+
+    // TODO Change placeholder values to actual limits
+    private static final int forwardLimit = 100000;
```

Comment:
May want to consider naming these in caps to denote that they are constants when used. Also adding TICKS to the name would clarify what limit they represent.